thirdparty/build-thirdparty.sh

```diff
@@ -80,6 +80,18 @@ check_prerequest() {
     fi
 }
 
+# echo if gcc version is greater than 14.0.0
+# else echo ""
+echo_gt_gcc14() {
+    local version=$($CC --version | grep -oP '(?<=\s)\d+\.\d+\.\d+' | head -1)
+    if [[ $(echo -e "14.0.0\n$version" | sort -V | tail -1) == "14.0.0" ]]; then
```

Comment:
`sort -V` is not portable

Reply:
It's OK for extracted versions from grep.